vaadin-testbench-shared/src/main/java/com/vaadin/testbench/parallel/setup/LocalDriver.java

```diff
@@ -71,12 +71,14 @@ static public WebDriver createDriver(
             }
             driver = new FirefoxDriver(options);
         } else if (BrowserUtil.isChrome(desiredCapabilities)) {
+            ChromeOptions options = new ChromeOptions();
             // Tells chrome not to show warning
             // "You are using an unsupported command-line flag:
             // --ignore-certifcate-errors".
             // #14319
-            ChromeOptions options = new ChromeOptions();
             options.addArguments("--test-type ");
+            // Disable search engine choice screen
+            options.addArguments("--disable-search-engine-choice-screen");
```

Comment:
I'm not sure if the search engine selection should be disabled for other types of tests too. 

However for local tests it's problematic because it seems that you can't customize the Chrome options in your test code - the method here ignores just ignores the capabilities passed in, so even if you would try to add this option in your test setup it doesn't work.